src/openapi/streaming/streaming.js

```diff
@@ -473,6 +473,17 @@ Streaming.prototype.subscribe = function(subscription) {
 };
 
 /**
+ * Makes a subscription start with modification.
+ * Modify subscription will keep pending unsubscribe followed by modify subscribe.
+ *
+ * @param {saxo.openapi.StreamingSubscription} subscription - The subscription to modify.
+ */
+Streaming.prototype.modify = function(subscription) {
+
+	subscription.onSubscribe(true);
```

Comment:
As per previous comment, I was expecting this to look like
```
modify(subscription, args) {

    subscription.onModify(args);
}
```
later an optional argument "options"
and
```
onModify(args) {
   this.onUnsubscribe();
   this.args = args;
    this.onSubscribe(true);
}
```

reasoning - otherwise we are fiddling with the classes own args (which it uses itself to auto subscribe), so I want to hand the whole modify process (including later patch option) to the library.